src/Bicep.Core.Samples/Files/Completions/declarations.json

```diff
@@ -167,6 +167,204 @@
       "newText": "param ${1:Identifier} string {\n  secure: true\n}"
     }
   },
+  {
```

Comment:
I'm confused.  What are these for, they appear to be duplicates of the snippet files.

Reply:
They're part of our baseline test infra. It's a serialized and normalized form of the LSP `CompletionItem`. Not particularly useful on something hard-coded like these snippets, but it's been helpful to prevent regressions in other completion areas.

---
In reply to: [595622862](https://github.com/Azure/bicep/pull/1904#discussion_r595622862) [](ancestors = 595622862)

Reply:
Thx.  Didn't see anything in the path that suggested tests.

Reply:
Yeah, "Samples" is kind of ambiguous naming-wise, but that project includes both user-committed bicep conversions as well as inputs into most of our data-driven tests.

---
In reply to: [595624827](https://github.com/Azure/bicep/pull/1904#discussion_r595624827) [](ancestors = 595624827)